lib/plugins/output-filter/journald-format.js

```diff
@@ -134,7 +134,7 @@ function applySematextCommonSchema (
             Object.assign(log[type], data)
           }
           callback(null, log)
-        }.bind(this)
+        }
```

Comment:
I have no idea what the bind() did here. Is it easy to explain? :)

Reply:
Sure. The `bind()` method is a prototype method on the function prototype chain in JavaScript, that returns a new function with a bound scope. It binds the `this` context into the bound function. There is no outer scope to the `this` which is referred here, hence making it redundant.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind):
> The `bind()` method creates a new function that, when called, has its this keyword set to the provided value, with a given sequence of arguments preceding any provided when the new function is called.